### ISSUE TITLE
incorrect architecture

### ISSUE THREAD

**Issue by nivkyard** [acids-ircam/nn_tilde#41]
Hi,
This project sound really promissing, but i can't use nn~ object.
I'm sorry i read the instruction on compilling stuff... but i'm new hear and don't understand how to do it.
Basically when i try to call nn~ in max it say "incorrect architecture". On macos 10.13.
What i have to do? 
Please help me, haha.
Thanks

**Comment by caillonantoine** [comment]
Hi,

Are you using a Silicon or intel mac ? (M1-M2 or old intel based ?)

**Comment by Abbienew** [comment]
I am, is there any difference?

**Comment by carloscastellanos** [comment]
I am having the same problem. I am on an Intel Mac, macOS 12.6.2. I downloaded x64 zip (https://github.com/acids-ircam/nn_tilde/releases/download/v1.5.6/nn_max_msp_macOS_x64.tar.gz) and when I try to load it in Max (8.3.1) it gives me the "incorrect architecture" error. What's even more puzzling is I was able to install and run it just fine on different Intel Mac (with the same OS version). I literally used the same zip file for both machines.
